src/pages/resources/sdks-end-of-support.md

```diff
@@ -2,6 +2,10 @@
 
 ## End of support for Adobe Experience Platform 1.x SDKs for Android
 
+**Updated On:July 8, 2024**
```

Comment:
Minor nit - needs a space between the colon and the July 8. So "Updated on: July 8, 2024".